secure-mobile-push/smp-android-client/app/src/main/res/values/strings.xml

```diff
@@ -1,3 +1,6 @@
 <resources>
     <string name="app_name">Secure Mobile Push</string>
+    <string name="send_a_sms_for_app_to_read">Send a SMS for the App to Read</string>
```

Comment:
Fixed in commit: https://github.com/adorsys/secure-key-storage/commit/87a2a2c02f2d278a60065555e93e83c276326ff5